src/ontology/edits/oeo-shared.omn

```diff
@@ -274,6 +277,43 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/538",
         <http://purl.obolibrary.org/obo/BFO_0000148>
     
     
+Class: OEO_00050016
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Final energy consumption is the consumption of energy delivered to and consumed by end users."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/576
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/613",
+        rdfs:label "final energy consumption"@en
+    
+    SubClassOf: 
+        OEO_00140039
+    
+    
+Class: OEO_00050017
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Gross inland consumption is the total consumption of energy in a geographical region (e.g. a country)."@en,
```

Comment:
"spatial" instead of "geographical" to use coherent terms?! Same question below.
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Gross inland consumption is the total consumption of energy in a spatial region (e.g. a country)."@en,
```